diff --git a/node_modules/@here/harp-webpack-utils/scripts/HarpWebpackConfig.js b/node_modules/@here/harp-webpack-utils/scripts/HarpWebpackConfig.js
index da55818..b233535 100644
--- a/node_modules/@here/harp-webpack-utils/scripts/HarpWebpackConfig.js
+++ b/node_modules/@here/harp-webpack-utils/scripts/HarpWebpackConfig.js
@@ -95,7 +95,7 @@ function createPlugins(htmlTemplate) {
         new CopyWebpackPlugin({
             patterns: [
                 {
-                    from: "node_modules/@here/harp-map-theme/resources",
+                    from: "../../node_modules/@here/harp-map-theme/resources",
                     to: "resources",
                     toType: "dir"
                 }
